packages/components/src/components/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,305 @@
+  })
+
+  describe('Warning variant', () => {
+    describe('Light mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'light')
+
+        component = render(
+          <Alert variant="warning" header={headerText} testID={testID} />,
+        )
+
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#faf3d1')
+        expect(borderLeftColor).toBe('#ffbe2e')
+        expect(getTextColor(component)).toBe('#3d4551')
+        expect(getIconName(component)).toBe('ExclamationTriangle')
+      })
+    })
+
+    describe('Dark mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'dark')
+        component = render(
+          <Alert variant="warning" header={headerText} testID={testID} />,
+        )
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#5c4809')
+        expect(borderLeftColor).toBe('#face00')
+        expect(getIconName(component)).toBe('ExclamationTriangle')
+        expect(getTextColor(component)).toBe('#f0f0f0')
+      })
+    })
+  })
+
+  describe('Error variant', () => {
+    describe('Light mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'light')
+
+        component = render(
+          <Alert variant="error" header={headerText} testID={testID} />,
+        )
+
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#f8dfe2')
+        expect(borderLeftColor).toBe('#b50909')
+        expect(getTextColor(component)).toBe('#3d4551')
+        expect(getIconName(component)).toBe('ExclamationCircle')
+      })
+    })
+
+    describe('Dark mode', () => {
+      it('should render correct colors and icon', () => {
+        mockedColorScheme.mockImplementation(() => 'dark')
+        component = render(
+          <Alert variant="error" header={headerText} testID={testID} />,
+        )
+        const { backgroundColor, borderLeftColor } = getContentStyle(component)
+        expect(backgroundColor).toBe('#5c1111')
+        expect(borderLeftColor).toBe('#d83933')
+        expect(getIconName(component)).toBe('ExclamationCircle')
+        expect(getTextColor(component)).toBe('#f0f0f0')
+      })
+    })
+  })
+
+  describe('Expandable variant', () => {
+    describe('Initialize collapsed', () => {
+      beforeEach(() => {
+        component = render(
+          <Alert
+            variant="error"
+            header={headerText}
+            description={descriptionText}
+            expandable
+            primaryButton={primaryButtonProps}
+            secondaryButton={secondaryButtonProps}>
+            {children}
+          </Alert>,
+        )
+      })
+
+      it('should initially render header only', async () => {
```

Comment:
```suggestion
      it('should initially render header only then render everything after press', async () => {
```
or something to that effect, doesn't adequately describe this test kind of doing double duty.